go/oasis-node/cmd/common/metrics/metrics.go

```diff
@@ -272,6 +272,18 @@ func GetDefaultPushLabels(ti *env.TestInstanceInfo) map[string]string {
 		for k, v := range flags.GetStringMapString(CfgMetricsLabels) {
 			labels[k] = v
 		}
+
+		// Remove empty label values - workaround for
+		// https://github.com/prometheus/pushgateway/issues/344
```

Comment:
File issue to remove this workaround once fixed upstream.

Reply:
Opened https://github.com/oasislabs/oasis-core/issues/2899